masonite/session.py

```diff
@@ -0,0 +1,47 @@
+
+
+class Session():
+    """
+    Class Session for manage sessions of the app
+    """
+    _session = {}
+
+    def __init__(self, environ):
+        """
+        Constructor
+        """
+        self.environ = environ
+
+    def get(self, key):
+        """
+        Get a session from object _session
+        """
+        ip = self.__get_client_address()
+        try:
+            session = self._session[ip][key]
+        except KeyError:
+            session = ''
+
+        return session
+
+    def set(self, key, value):
+        """
+        Set a new session in object _session
+        """
+        ip = self.__get_client_address()
+        self._session[ip] = {key: value}
+
+    def reset(self):
```

Comment:
This should be good. Um lets also also do 2 more things here. We need a way to:

1. Reset all session data from a certain IP address.

So you can just specify a second parameter:

```python
def reset(self, string ip=None, bool current_user=False):
```

then just do those actions so if the delete from the the ip key if one is passed or delete from the current user (using your private method of getting the IP address) if that is True

The `string` and `bool` prefixes are just for you so you get an idea of what should be passed in.